Packages/io.chainsafe.web3-unity/Editor/ServerSettings.cs

```diff
@@ -204,4 +208,25 @@ private class ValidateProjectIDResponse
         [JsonProperty("response")]
         public bool Response { get; set; }
     }
+    
+    static void WebGLTemplateCheck()
+    { 
+        // Only checks the templates and prompts for update if we're in webgl
+        #if UNITY_WEBGL
+        var performSync = WebGLTemplateSync.CheckSyncStatus() switch
+        {
+            WebGLTemplateSyncStatus.UpToDate => false,
+            WebGLTemplateSyncStatus.DoesntExist =>
+                EditorUtility.DisplayDialog("web3.unity", "Do you wish to install the web3.unity WebGL templates into your project?", "Yes", "No"),
+            WebGLTemplateSyncStatus.OutOfDate =>
+                EditorUtility.DisplayDialog("web3.unity", "The web3.unity WebGL templates in your project are out of date, would you like to update now?", "Yes", "No"),
+            _ => false,
+        };
+        
+        if (performSync)
+        {
+            WebGLTemplateSync.Syncronize();
```

Comment:
Typo (Though I dont see that function within this PR so might be a separate bug fix

```suggestion
            WebGLTemplateSync.Synchronize();
```